js/simple-pig-latin.js

```diff
@@ -0,0 +1,23 @@
+// https://www.codewars.com/kata/520b9d2ad5c005041100000f/train/javascript
+
+function pigIt(str){
+  const wordsArray = str.split(' ');
+  const resultArray = wordsArray.map((x, i) => {
+      const format = /[ !@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?]/;
+      let arr = x.split('');
```

Comment:
1. Do you need to move the first letter to the end of the world if it's just punctuation?
2. How about extracting the operation of moving the first letter to the end to a separate function?